### PR TITLE
feat: #1344 update readme mw-webapp

### PR DESCRIPTION
**Pull request by AKiroi**
update README.md in mw-webapp

### CODE REVIEW

**Review by tritonJS826 — CHANGES_REQUESTED** [review]
Looks like duplication of root readme, why do we need it?

File: mw-webapp/README.md [hunk 1]
```diff
@@ -1,6 +1,126 @@
...
+
+- [Node.js](https://nodejs.org/) v22.5.1 (we recommend using [nvm](https://github.com/nvm-sh/nvm) or [n](https://github.com/tj/n) for installation)
+- [pnpm](https://pnpm.io/) v8.15.2
+- [Vite](https://vitejs.dev/) v5.4.1
+- [TypeScript](https://www.typescriptlang.org/) v5.3.3
+- [React](https://reactjs.org/) v18.2.0
+
+Additional tools:
+- [Docker](https://www.docker.com/) v24.0.7
+- [Docker Compose](https://docs.docker.com/compose/) v1.29.2
```
Comment (tritonJS826) [inline mw-webapp/README.md:15]:
```suggestion
```

File: mw-webapp/README.md [hunk 2]
```diff
@@ -1,6 +1,113 @@
...
+   ```sh
+   docker-compose -f local.docker-compose.yml up
+   ```
+
+
+## Start project 
+  - Run:
+   ```sh
+  pnpm run start
+   ```
```
Comment (tritonJS826) [inline mw-webapp/README.md:33]:
```suggestion
```

File: mw-webapp/README.md [hunk 3]
```diff
@@ -1,6 +1,83 @@
...
+- Run unit tests:
+  ```sh
+  pnpm test
+  ```
+
+- Run component tests:
+  ```sh
+  pnpm cypress:component:run
+  ```
+
```
Comment (tritonJS826) [inline mw-webapp/README.md:16]:
```suggestion
- Run component tests:
  ```sh
  pnpm cypress:component:run
  ```

- Run component tests ui mode:
  ```sh
  pnpm cypress:component:open
  ```

```

File: mw-webapp/README.md [hunk 4]
```diff
@@ -1,6 +1,81 @@
...
+  pnpm cypress:component:run
+- Run end-to-end tests:
+  ```sh
+  pnpm cypress:e2e:run
+  ```
+
+- Run all tests:
+  ```sh
+  pnpm test:module
+  ```
```
Comment (tritonJS826) [inline mw-webapp/README.md:23]:
```suggestion
- Run component tests:
  ```sh
  pnpm cypress:component:run
  ``` 
- Run component tests ui mode:
  ```sh
  pnpm cypress:component:open
  ``` 

- Run end-to-end tests:
  ```sh
  pnpm cypress:e2e:run
  ```
 - Run end-to-end tests ui mode:
  ```sh
  pnpm cypress:e2e:open
  ```
```

File: mw-webapp/README.md [hunk 5]
```diff
@@ -1,6 +1,81 @@
...
+
+## API Generation
+
+To generate API clients:
+
+```sh
+pnpm api:generate
+```
+
```
Comment (tritonJS826) [inline mw-webapp/README.md:59]: add all and only module-specific scripts (cypress:open, serve etc)

File: mw-webapp/README.md [hunk 6]
```diff
@@ -1,6 +1,81 @@
...
+## Testing
+
+- Run unit tests:
+  ```sh
+  pnpm test
+  ```
+
+- Run component tests:
+  ```sh
+  pnpm cypress:component:run
```
Comment (tritonJS826) [inline mw-webapp/README.md:14]: sorry I broke it with github suggestion :)

**Comment by Ekaterina1994** [comment]
Need to add info that command pnpm run api:generate works only with pnpm serve, with pnpm start it doesn't works

**Pull request closed without merging**